functions/countdown-timer.ts

```diff
@@ -0,0 +1,56 @@
+import { Handler, HandlerEvent } from '@netlify/functions';
+import fetch from 'node-fetch';
+import { DateTime } from 'luxon';
+
+const months = [
```

Comment:
We have some date utils already and you could use `import { month } from './utils/date-utils';` here